tests/configDirectory/php-cs-fixer.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+/*
+ * This file is part of the box project.
+ *
+ * (c) Anatoliy Melnikov <5785276@gmail.com>
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+use Aeliot\PhpCsFixerBaseline\Service\FilterFactory;
+
+$rules = [
+    '@Symfony' => true,
+    '@Symfony:risky' => true,
+    'concat_space' => [
+        'spacing' => 'one',
+    ],
+    'header_comment' => [
+        'header' => <<<'EOF'
+            This file is part of the box project.
+
+            (c) Anatoliy Melnikov <5785276@gmail.com>
+
+            This source file is subject to the MIT license that is bundled
+            with this source code in the file LICENSE.
+            EOF,
+    ],
+    'phpdoc_align' => ['align' => 'left'],
+];
+
+/** @var PhpCsFixer\Config $config */
+$config = (new PhpCsFixer\Config())
+    ->setRiskyAllowed(true)
+    ->setRules($rules);
+
+$configReader = Aeliot\PhpCsFixerBaseline\Service\ConfigReader::getInstance();
```

Comment:
Why do you try to parse option used during the calling of `vendor/bin/php-cs-fixer`? It have neither option `--dir` no `-d`. More over in the config file you may easily define path correct path to baseline which belongs the this config without any console option.

I thought you wont to use option `--dir` for autodetection of required config, finder and baseline during the building of baseline. And not during the calling of fixer.